src/main/java/mainClass/Main.java

```diff
@@ -1,3 +1,5 @@
+package mainClass;
```

Comment:
**quibble**: Everything in java is a class 🙃 

I think it would be acceptable to include `Main` in your `echoServer` package. When I am working with Java/Kotlin, I tend to keep things in a single package until I can clearly identify the boundaries and namespaces that I want to create.